backend/GymBeam/Controllers/UserController.cs

```diff
@@ -27,49 +28,59 @@ public UserController(IMediator mediator)
 #endif
         public async Task<ActionResult<IEnumerable<UserResponse>>> Get()
         {
-            var request = new GetAllUsersQuery();
-            var result = await _mediator.Send(request);
-            return Ok(result);
+            try
+            {
+                var request = new GetAllUsersQuery();
+                var result = await _mediator.Send(request);
+                return Ok(result);
+            }
+            catch (DomainException ex)
```

Comment:
what if somehow there will be diffrent exception thrown? We need to catch it as well